main.tf

```diff
@@ -12,6 +12,17 @@ resource "aws_instance" "this" {
     volume_size = 1000
     iops        = 800
   }
+  monitoring           = true
+  iam_instance_profile = "<valid_iam_role>"
+
```

Comment:
**Tenable.cs remediation**
 
**[LOW]: Ensure tracing is enabled for AWS Lambda Functions**
 
**Vulnerability**
Lambda tracing if not enabled can have an impact the integrity and incident response.
 
**Remediation**
In AWS Console - 
 1. Sign in to AWS Console and go to the Functions page on the Lambda console.
 2. Select a function.
 3. Select Configuration and then choose Monitoring tools.
 4. Select Edit.
 5. Under X-Ray, enable Active tracing.
 6. Click Save.

source: tenablecs-